velox/dwio/parquet/tests/reader/ParquetReaderTest.cpp

```diff
@@ -16,14 +16,31 @@
 
 #include "velox/dwio/parquet/reader/ParquetReader.h"
 #include "velox/dwio/parquet/tests/ParquetReaderTestBase.h"
+#include "velox/expression/ExprToSubfieldFilter.h"
 
 using namespace facebook::velox;
 using namespace facebook::velox::common;
 using namespace facebook::velox::dwio::common;
 using namespace facebook::velox::dwio::parquet;
 using namespace facebook::velox::parquet;
 
-class ParquetReaderTest : public ParquetReaderTestBase {};
+class ParquetReaderTest : public ParquetReaderTestBase {
+ public:
+  void assertReadWithFilters(
+      const std::string& fileName,
```

Comment:
fileName is always the same. Consider removing this parameter to make test code shorter. Same for fileSchema.